src/components/FormSearch/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import IconSearch from "../../icons/IconSearch";
+import "./style.css";
+import IconMail from "../../icons/IconMail";
+const FormSearch = ({ plaseholder, icon }) => {
```

Comment:
you can write this component like this (without return, becouse return including inside arrow function if we dont use this `{``}`)https://learn.javascript.ru/es-function#funktsii-cherez
```
const FormSearch = ({ plaseholder, icon }) => (
    <form className="search-form">
      <input
        className="search-form__input"
        type="text"
        placeholder={plaseholder}
      />
      <button className="search-form__button" type="submit">
        {icon === "search" && <IconSearch />}
        {icon === "mail" && <IconMail />}
      </button>
    </form>
  )
```